app/models/user.rb

```diff
@@ -1,6 +1,10 @@
 class User < ApplicationRecord
-  has_many :author_id, foreign_key: 'author_id'
-  has_many :user_id, foreign_key: 'user_id'
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable, :confirmable,
+         :recoverable, :rememberable, :validatable
```

Comment:
* According to the [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md#:~:text=Ask%20for%20confirmation,Can%20reset%20password.) these should work 
`Can reset password.` but currently I can not reset my password below is a recording of the said issue 

https://github.com/Zven94/BlogApp/assets/52497246/3a666c4f-85d0-45f1-a7e8-9112239034ef

Reply:
@Davidosky007 Thank you very much for testing my app. But I just test it and the reset is working with no problem. Hereyou can see the screenshots. I'm going to work in the other changes anyway. Thank you

**Console with the link to reset:**
![reset_password](https://github.com/Zven94/BlogApp/assets/125831572/cc7689ff-5601-4d33-a12f-8c18f2946527)

SS of the view:
![reset_password_view](https://github.com/Zven94/BlogApp/assets/125831572/323cc6bc-adfd-41d8-a152-186185c747d8)

Reply:
Hi @Davidosky007 thank you very much for your review. I just test the password reset and it's working fine in my app. Here you can see the successful msg on my console and I changed nothing in my the repo.

![reset_password_succesful](https://github.com/Zven94/BlogApp/assets/125831572/daf1d468-7fc9-4b75-b772-3cea54de401b)